xml/System.Runtime.InteropServices/ComWrappers.xml

```diff
@@ -230,7 +230,7 @@ If the object cannot be created and `null` is returned, the call to <xref:System
         <param name="instance">The managed object to expose outside the .NET runtime.</param>
         <param name="flags">Flags used to configure the generated interface.</param>
         <summary>Creates a COM representation of the supplied object that can be passed to a non-managed environment.</summary>
-        <returns>The generated COM interface that can be passed outside the .NET runtime.</returns>
+        <returns>The generated COM interface, set to the <code>IUnknown</code> vtable, that can be passed outside the .NET runtime.</returns>
```

Comment:
```suggestion
        <returns>The generated COM interface, as a pointer to the <code>IUnknown</code> interface implementation, that can be passed outside the .NET runtime.</returns>
```

I don't like the wording of "pointer to the vtable" since it's a pointer to the COM object, not the vtable itself.

Reply:
Great point. Thanks.